docs/oauth2.md

```diff
@@ -538,13 +664,94 @@ In the following RAR example
 }
 ```
 
-if RabbitMQ node's `resource_server_id` is equal to `finance`, the plugin will compute the following sets of scopes:
+if RabbitMQ nodes `resource_server_id` is equal to `finance`, the plugin will compute the following sets of scopes:
 
 - `finance.read:primary-*/*/*`
 - `finance.write:primary-*/*/*`
 - `finance.configure:primary-*/*/*`
 - `finance.tag:administrator`
 
+
+
+## Advanced usage {#advanced-usage}
+
+### Use default OAuth 2.0 provider {#use-oauth-provider}
+
+As long you have one and only one OAuth 2.0 provider, you can skip this advanced usage although you can use it.
```

Comment:
Do we need to say "As long as you have one and only one" how about "As long as you have only one OAuth 2.0 provider, ..."